<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDialog</class>
 <widget class="QDialog" name="mainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hugh Stock Week</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="loginBtn">
        <property name="text">
         <string>LOGIN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>START</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>PAUSE</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="runningLabel">
        <property name="text">
         <string>상태</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="activateCB">
        <property name="text">
         <string>매매</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settingBtn">
        <property name="text">
         <string>설정</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="buyBtn">
        <property name="text">
         <string>매수전략</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="buyLabel">
        <property name="text">
         <string>BuyLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sellBtn">
        <property name="text">
         <string>매도전략</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sellLabel">
        <property name="text">
         <string>SellLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="manualBtn">
        <property name="text">
         <string>수동매매</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPlainTextEdit" name="logEditText">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
